standard/attributes.md

```diff
@@ -805,11 +805,17 @@ Caller information is only substituted when a function is explicitly invoked in
 
 One exception is query expressions. These are considered syntactic expansions, and if the calls they expand to omit optional parameters with caller-info attributes, caller information will be substituted. The location used is the location of the query clause which the call was generated from.
 
-If more than one caller-info attribute is specified on a given parameter, they are preferred in the following order: `CallerLineNumber`, `CallerFilePath`, `CallerMemberName`.
+If more than one caller-info attribute is specified on a given parameter, they are recognized in the following order: `CallerLineNumber`, `CallerFilePath`, `CallerMemberName`. Consider the following parameter declaration:
+
+```csharp
+[CallerMemberName][CallerFilePath][CallerLineNumber]object p = ...
```

Comment:
Nit: can we have a space after the final `]`? And I'd personally write this as:

```csharp
[CallerMemberName, CallerFilePath, CallerLineNumber] object p = ...
```

... but I'm happy to stick to separating them entirely if you think it's clearer.

Reply:
OK; will fix both of these.